### PR TITLE
Adding test to cover: Allowing a user to read metadata from project w…

### PR DESCRIPTION
**Pull request by carvilsi**
Adding test to cover: Allowing a user to read metadata from project without administration access for project

I certify that:
- [x] The automated build passes on the branch this pull request is from
- [x] My change is documented in the Changelist.md
- [x] Any functionality additions/changes are documented in the README.md
- [x] Any new dockerimages this build depends on are properly tagged on github
- [x] I shall not merge until the travis build is green and the PR is accepted by
  another developer
- [x] I shall promptly fix/reply to comments, as I know PRs are not :fire: and
  forget
- [x] I shall keep my branch updated with any intermediate changes to the
  target branch
- [x] This is a quality PR
- [x] I got to the end of this list

### CODE REVIEW

File: Changelist.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelist
 
+## 7.3.2
```
Comment (gvdent) [inline Changelist.md]: Adding tests doesn't require a new version or Changelist.md entry, they are not meant for the users of the SDK.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "weaver-sdk",
-  "version": "7.3.1",
+  "version": "7.3.2",
```
Comment (gvdent) [inline package.json]: Please don't update to non-beta versions in feature branches. _Always_ add -beta.x so no versions get published accidentally.

That and this doesn't change the SDK in anyway, so no version update is required.

File: Changelist.md (hunk 1 above)
Comment (carvilsi) [reply Changelist.md]: 👍

File: package.json (hunk 2 above)
Comment (carvilsi) [reply package.json]: 👍

**Pull request merged**